### ISSUE TITLE
About test acc

### ISSUE THREAD

**Issue by Xxiaoluohao** [lulutang0608/Point-BERT#18]
I noticed that you used farthest point sample in the testing phase, while others simply took the first 1024 points, or picked them at random. I tried to train a DGCNN classification model with farthest point sample and achieved 82.6 on the ScanObjectNN(PB T50 RS) test set, which is much higher than 78.1 for random sample. So I would like to know which method I should use to calculate the acc?

**Comment by yuxumin** [comment]
Hi, 

It is strange to know this. *random sampling* and *fps*  are two main methods for point cloud downsampling. Are you sure the performance gap comes from different downsampling methods?  (the results of 78.1 for DGCNN comes from official leadboard, using 2048 points. do i mistake something? )

**Comment by Xxiaoluohao** [comment]
The results of 78.1 comes from [scanobjectn](https://github.com/hkust-vgd/scanobjectnn), using 1024 points and random sample. I first use your classification code to train this [model](https://github.com/AnTao97/dgcnn.pytorch/blob/master/model.py), and got 82.68, then I changed the fps to random sampling and the result dropped to 79.39. In fact, using the model trained by the former, only changing the sampling method of the test set will also see a significant performance drop.

**Comment by yuxumin** [comment]
Hi,
I test the pretrained model under `random sampling` method.
![pic_scanobjectnn](https://user-images.githubusercontent.com/29698233/150918756-5e5d7aca-1e8a-4fd4-ba48-f17594b8e378.png)

```
def random_choose(data, number):
    new_data = []
    for cloud in data:
        idx = torch.randperm(cloud.size(0))
        new_data.append(cloud[idx[:number]])
    return torch.stack(new_data)
```
 The performance is about 82 acc. I think it is a reasonable result. Because we choose the best model for pointBERT by testing it on fps data. And i think it can provide another evidence that our Transformer pretrained by MPM is more robust to the real-world data. (`random sampling` can be regarded as an augmentation for data containing 1024 points)
As for test, we recommend to use fps data for reproduction.

**Comment by Xxiaoluohao** [comment]
Thanks for your clarification.
By the way, I am also trying to reproduce the segmentation task, could you please provide me the [seg head model code](https://github.com/lulutang0608/Point-BERT/issues/17) conveniently?

**Comment by yuxumin** [comment]
Sure, plz email me and then i will reply you.

**Comment by Xxiaoluohao** [comment]
> Sure, plz email me and then i will reply you.

Thank you so much! My email is zxluo@mail.ustc.edu.cn.
